power_grid_model_c/power_grid_model/include/power_grid_model/auxiliary/serialization/serializer.hpp

```diff
@@ -19,6 +19,20 @@
 #include <span>
 #include <string_view>
 
+constexpr void pack_inf(auto& packer_inf, auto& attr) {
```

Comment:
since this function is quite specific, please be explicit

```suggestion
constexpr void pack_inf(msgpack::packer<Stream>& p packer_inf, std::floating_point auto& attr) {
```